Dockerfile

```diff
@@ -300,7 +287,7 @@ COPY --from=swagger       /build/ /usr/local/bin/
 COPY --from=tomlv         /build/ /usr/local/bin/
 COPY --from=tini          /build/ /usr/local/bin/
 COPY --from=registry      /build/ /usr/local/bin/
-COPY --from=criu          /build/ /usr/local/
+COPY --from=criu          /usr/sbin/criu /usr/local/bin
```

Comment:
I guess `apt-get install` does not have a `--prefix` option (unless compiling form source) 😓 (would've been nice if we could keep the same paths)

Reply:
We could do something cute like `criu="$(which criu)"; ln "$criu" /build/` after the `apt-get install` :trollface:

Reply:
Given that we don't run the intermediate `criu` stage, and only use it to download the package, I took a different approach in https://github.com/moby/moby/pull/41739/files#r634478731

Reply:
We can literally have images in ghcr or hub for all of these things we currently build in the Dockerfile..
Speed things up a lot.

Reply:
But then we'd have to maintain (multi-arch) images for each of those stages?

Reply:
Shouldn't be terribly difficult.
We only need to add the versions we pull in here.